app/views/articles/_actions.html.erb

```diff
@@ -30,7 +30,7 @@
     <div id="mod-actions-menu-btn-area" class="print-hidden hidden align-center"></div>
     <div class="align-center m:relative">
       <button id="article-show-more-button" aria-controls="article-show-more-dropdown" aria-expanded="false" aria-haspopup="true" class="dropbtn crayons-btn crayons-btn--ghost-dimmed crayons-btn--icon-rounded" aria-label="<%= t("views.actions.more.aria_label") %>">
-        <%= inline_svg_tag("overflow-horizontal.svg", aria_hidden: true, class: "dropdown-icon crayons-icon", title: t("views.actions.more.title")) %>
+        <%= crayons_icon_tag("overflow-horizontal", css_class: "dropdown-icon", title: t("views.actions.more.title")) %>
```

Comment:
It doesn't have to be for this PR, but we should update the `crayons_icon_tag` helper to allow us to set `aria_hidden: true`, since we're losing that in a few places here.  In a lot of cases, it's better for screen reader users to hide the icon, since the `title` attribute isn't consistently surfaced and can also result in funky label names

Reply:
Eek, that's just my fault, I guess I mistook them for `aria: true` which the helper adds. We should be able to pass in `aria_hidden: true` as part of the `**options`, I'll review this PR one more time myself.

Reply:
@aitchiss I re-added the lost `aria_hidden: true` attributes now.